test/common/router/scoped_rds_test.cc

```diff
@@ -6,16 +6,22 @@
 
```

Comment:
I suggest also validating `ScopedRdsConfigSubscription::scopedRouteMap()` and `ScopedRdsConfigProvider::getConfig()` where applicable in these tests.

Reply:
done